packages/default/scss/dock-manager/_layout.scss

```diff
@@ -0,0 +1,231 @@
+@mixin kendo-dock-manager--layout-base() {
+
+    // DockManager
+    .k-dock-manager {
+        display: flex;
+        flex-flow: row nowrap;
+        position: relative;
+        border-width: $kendo-dock-manager-border-width;
+        border-style: $kendo-dock-manager-border-style;
+
+        .k-pane,
+        .k-splitbar {
+            box-sizing: border-box;
+        }
+
+        .k-pane-flex {
+            flex-direction: column;
+        }
+
+        .k-splitbar-horizontal {
+            border-inline-width: $kendo-dock-manager-splitbar-border-width;
+            border-inline-style: $kendo-dock-manager-splitbar-border-style;
+        }
+
+        .k-splitbar-vertical {
+            border-block-width: $kendo-dock-manager-splitbar-border-width;
+            border-block-style: $kendo-dock-manager-splitbar-border-style;
+        }
+
+        .k-tabstrip-content,
+        .k-splitter {
+            border-width: 0;
+        }
+
+        .k-toolbar .k-separator-horizontal {
+            width:  $kendo-line-height-em;
+            height: 0;
+            border-width: $kendo-dock-manager-separator-border-width 0 0;
+        }
+
+    }
+
+        // Needed because of a bug in Chrome - Issue 1473554
+        // A fix will be shipped likely in milestone 118 and then we can revisit this
+        @supports  (not (-ms-ime-mode: none)) and (not (overflow: -webkit-marquee)) and (not (-moz-appearance:none)) {
+            .k-dock-manager-toolbar .k-toolbar-button {
+                padding-block: $kendo-dock-manager-toolbar-button-padding-y;
+                padding-inline: $kendo-dock-manager-toolbar-button-padding-x;
```

Comment:
We need the variables for the time being otherwise the buttons will not be the correct size -- they do not share the same padding across all themes. What I've done now is ommit the `!default` option so they are not customizable since we won't be keeping these styles for long. Let me know what you think.